src/borg/archive.py

```diff
@@ -691,6 +691,7 @@ def restore_attrs(self, path, item, symlink=False, fd=None):
                 if e.errno == errno.E2BIG:
                     logger.warning('%s: Value or key of extended attribute %s is too big for this filesystem' %
                                    (path, k.decode()))
+                    set_ec(EXIT_WARNING)
```

Comment:
not directly related to your previous work, but for consistency: in 695 it basically ignores ENOTSUP and EACCES (sic).

ENOTSUP: if there is not xattr support at all (== even worse than only supporting small to medium xattrs), it stays silent (no warning log entry) and does not set warning rc.

EACCES: not sure about that one but sounds like worth a warning at least too and not silently ignoring it.
We should discuss about that one though, I think that could be maybe borg running as normal user with no permissions to set security related stuff (?). OTOH, we have special ACL support for posix acls. Not sure what the issue was with EACCES and security.*. For normal file mode (st_mode) and user/group, I think we currently also silently ignore them when we do not run as root, but just as a normal user who can not create some mode and files for other users/groups.